src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.suppliers;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]);
```

Comment:
```suggestion
        return Color.values()[index].name();
```
if you have questions or need any help, please write in jv_chat
because mentors don't see those questions